.github/workflows/ci.yaml

```diff
@@ -90,6 +90,7 @@ jobs:
               - "coderd/**"
               - "enterprise/**"
               - "examples/*"
+              - "helm/**"
```

Comment:
@ethanndickson does that look right to you?

Reply:
Yep